src/components/Calendar.tsx

```diff
@@ -16,6 +16,8 @@ export type CalendarProps = {
   timeStart?: string;
   /** Limit the view to appointments before this time */
   timeEnd?: string;
+  /** Number of days in a view */
+  customDays: number;
```

Comment:
The default value is very good, but maybe we can make it 4? The week view is already 7 so I think that would make more sense. You would also have to make it optional here with a `?`, so that it is not required by TypeScript.

Reply:
fixed